run.py

```diff
@@ -905,6 +906,99 @@ def hvreads(args):
             "aws", "s3", "cp", output, "%s/%s/hvreads/%s" % (
                S3_BUCKET, args.bioproject, output)])
 
+def alignments(args):
+   available_inputs = get_files(args, "hvreads")
+   existing_outputs = get_files(args, "alignments")
+
+   for sample in get_samples(args):
+      output = "%s.alignments.tsv" % sample
+      if output in existing_outputs: continue
+
+      input_fname = "%s.hvreads.json" % sample
+      if input_fname not in available_inputs: continue
+
+      with tempdir("alignments", input_fname) as workdir:
+         subprocess.check_call([
+            "aws", "s3", "cp", "%s/%s/hvreads/%s" % (
+               S3_BUCKET, args.bioproject, input_fname), input_fname])
+
+         pair1 = []
+         pair2 = []
+         combined = []
+         with open(input_fname) as inf:
+            for seq_id, details in sorted(json.load(inf).items()):
+               if type(details[0]) == type(1):
+                  assignment = details.pop(0)
+               kraken_info, *reads = details
+               assert len(reads) in [1, 2]
+               if len(reads) == 1:
+                  read, = reads
+                  combined.append((seq_id, read))
+               else:
+                  read1, read2 = reads
+                  pair1.append((seq_id, read1))
+                  pair2.append((seq_id, read2))
+
+         for label, reads in [
+               ("pair1", pair1),
+               ("pair2", pair2),
+               ("combined", combined),
+         ]:
+            if reads:
+               with open("%s.fastq" % label, "w") as outf:
+                  for seq_id, read in reads:
+                     seq, quality = read
+                     outf.write("@%s\n%s\n+\n%s\n" % (seq_id, seq, quality))
+
+         sam_out = "out.sam"
+         cmd = [
+            "/home/ec2-user/bowtie2-2.5.2-linux-x86_64/bowtie2",
+            "--local",
+            "-x", "/home/ec2-user/bowtie/human-viruses",
```

Comment:
Is this a custom database or something that comes with the Bowtie2 install? If it's a custom db it would be good to add it to the dependencies section of the README

Reply:
Sorry, this is another thing @simonleandergrimm was adding, and is also in the now-merged https://github.com/naobservatory/mgs-pipeline/pull/35